bin/convert_finder_schemas

```diff
@@ -34,7 +33,16 @@ class DocumentTypeMapper
   }
 
   def call(filename)
-    MAPPING.fetch(File.basename(filename))
+    file_name = File.basename(filename)
+    begin
+      MAPPING.fetch(file_name)
+    rescue Exception => e
```

Comment:
The [style guide](https://github.com/alphagov/styleguides/blob/master/ruby.md#exceptions) discourages the use of exceptions for things like this. Rather, we can just look up the mapping, and check if a value is returned:

    document_type = MAPPING[filename]
    if document_type.nil?
       ....